docker/base-deps/Dockerfile

```diff
@@ -48,16 +46,22 @@ RUN sudo apt-get update -y && sudo apt-get upgrade -y \
         netbase \
         openssh-client \
         gnupg; fi) \
-    && wget \
-        --quiet "https://repo.anaconda.com/miniconda/Miniconda3-py37_23.1.0-1-Linux-${HOSTTYPE}.sh" \
+    && if [ "$PYTHON_VERSION" = "3.11" ]; then \
+            MINICONDA_VERSION="py310_23.3.1-0"; \
```

Comment:
nit: use `if [[ "PYTHON_VERSION" == "3.11" ]]; then`

also `PYTHON_VERSION` seems to have 3 number, as on line 14:

`PYTHON_VERSION=3.8.16`

so maybe you want to use `if [[ "${PYTHON_VERSION}" == "3.11" || "${PYTHON_VERSION}" =~ ^3\.11\. ]];`


or change line 14..